pkg/preference/preference.go

```diff
@@ -189,7 +199,7 @@ func (c *PreferenceInfo) SetConfiguration(parameter string, value string) error
 				return errors.Wrapf(err, "unable to set %s to %s", parameter, value)
 			}
 			if typedval < 0 {
-				return errors.Errorf("cannot set timeout to less than 0")
```

Comment:
why was this line removed?

Reply:
even linter is complaining ;-)
```
golangci-lint run ./... --timeout 5m
pkg/preference/preference.go:201:4: SA9003: empty branch (staticcheck)
			if typedval < 0 {
			^
make: *** [golint] Error 1
```

Reply:
Yikes, that's embarrassing, I'm not sure how I missed that or how it happened. But I've fixed it either way.